charts/kueue/values.yaml

```diff
@@ -150,9 +150,7 @@ webhookService:
       protocol: TCP
       targetPort: 9443
   type: ClusterIP
-
```

Comment:
nit, revert formatting changes of unrelated files

Reply:
Done